pom.xml

```
@@ -95,12 +95,15 @@
         specified. Used by the `patch` profile to enable patching. -->
         <error-prone.args />
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.auto-service>1.0-rc6</version.auto-service>
```

I assume we will upgrade the internal version soon.
```suggestion
        <version.auto-service>1.0-rc7</version.auto-service>
```